### PR TITLE
[Tokyo-2018] Add placeholder event details

### PR DESCRIPTION
**Pull request by BenLubar**
[Requesting review from @apapadimoulis before this gets merged]

### CODE REVIEW

File: data/events/2018-tokyo.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    employer: "Creationline"
+  - name: "Tadahiro Yasuda"
+    employer: "Creationline"
+
+organizer_email: "organizers-tokyo-2018@devopsdays.org" # Put your organizer email address here
+proposal_email: "proposals-tokyo-2018@devopsdays.org" # Put your proposal email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
```
Comment (bridgetkromhout) [inline data/events/2018-tokyo.yml]: Hey, @apapadimoulis - it looks like @BenLubar copied this file from last year. You probably want to verify that you're only listing sponsors who are actually signed on for 2018.

File: data/events/2018-tokyo.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  - name: program
+  - name: location
+    #- name: registration
+    #  url: https://www.eventbrite.com/e/devops-days-tokyo-2017-tickets-31764173498
+  - name: sponsor
+  - name: contact
+  - name: conduct
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (bridgetkromhout) [inline data/events/2018-tokyo.yml]: Hey, @apapadimoulis - can you verify that this is your 2018 team? I've set up the email list, but we should make adjustments if the team members have changed.

File: content/events/2018-tokyo/conduct.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
++++
+date = "2017-12-22T09:19:00-06:00"
+title = "conduct"
+type = "event"
+
++++
+
+## ANTI-HARASSMENT POLICY
```
Comment (bridgetkromhout) [inline content/events/2018-tokyo/conduct.md]: Would it make sense to provide a Japanese version of the code of conduct?

**Comment by bridgetkromhout** [comment]
Waiting for some input from @apapadimoulis before this is ready to be merged - thanks!

**Comment by bridgetkromhout** [comment]
Also noting that @BenLubar is not listed as an organizer for this event, so we do need 👍 from an organizer.

**Comment by apapadimoulis** [comment]
👍

**Pull request merged**